### ISSUE TITLE
Trying to port this to PIC mcu, hoping to talk

### ISSUE THREAD

**Issue by colinmarcus** [NordicSemiconductor/ble-sdk-arduino#5]
Hi, I'm trying to develop a port for the PIC24F series microcontrollers. I followed your instructions and tried to port your nRF8001 code, but I can't seem to get the transport layer verification to work. I've put in a lot of effort, but I feel that I've run into a wall - the problem is probably something really straightforward, but I just don't know your code well enough to see a solution.

To be specific, I am running:

ble_aci_transport_layer_verification.c

and the system can't seem to get past the 1st if statement:

void loop()
{
  // We enter the if statement only when there is a ACI event available to be processed
  if (lib_aci_event_get(&aci_state, &aci_data))
  {
      ...stuff
  }
}

In other words, the if statement never returns true. I have verified that the SPI seems to be running correctly, as well as all the other GPIOs, delays and so on. Interrupts are turned off.

I would really appreciate it if we could communicate.

My email is: c_marcus@mit.edu
